vue-app/src/store/index.ts

```diff
@@ -327,11 +328,17 @@ const actions = {
         }
       }
 
+      let ensName: string | null = state.currentUser.ensName
+      if (isAddress(state.currentUser.walletAddress)) {
```

Comment:
Can we safely assume this is an address at this point? If not, we probably should add this check sooner (e.g. before querying ether & token balances).

Reply:
Perhaps @pettinarip can confirm but I think we can safely assume this is a legit address. I think we only set `state.currentUser.walletAddress` from the Web3 plugin, which receives this from the connected accounts.

Reply:
Yeah, I think you're right... This is pulled from the wallet provider, will remove check